plaso/analyzers/hashers/entropy.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""The entropy calculation implementation."""
+
+from __future__ import division
+from __future__ import unicode_literals
+
+import collections
+import math
+
+from plaso.analyzers.hashers import interface
+from plaso.analyzers.hashers import manager
+
+
+class EntropyHasher(interface.BaseHasher):
```

Comment:
> EntropyHasher

ByteEntropyCalculator

Reply:
Leaving this as is, as the parent class here is called Hasher

Reply:
ack, as discussed